bip-0341.mediawiki

```diff
@@ -294,7 +294,15 @@ Examples of preimage for sighashing for each of the sighash modes.
 
 == Deployment ==
 
-TODO
+There may be multiple deployment attempts of this BIP.
+
+The first deployment will use BIP 8 with the name "taproot", using bit 2, and a lockinontimeout of false.
+
+For Bitcoin mainnet, the BIP 8 startheight will be 681408 (approximately May 1st 2021), the timeoutheight will be 695520 (approximately August 6th 2021), the minimum_activation_height will be 709632 (approximately November 12th 2021), and the activation threshold will be 1815 out of 2016 blocks (90%).
+
+For Bitcoin testnet, the BIP 8 startheight will be 1945440 (approximately April 20th 2021), The timeoutheight will be 1951488 (approximately June 1st 2021), the minimum_activation_height will be 1957536 (approximately July 13th 2021), and the activation threshold will be 1512 out of 2016 blocks (75%).
```

Comment:
The entire testnet range from block [1945440](https://blockstream.info/testnet/block/0000000000003531d7cc792e8fc02bfbbb701931f6a06751781a80329833c3d3) to [1957536](https://blockstream.info/testnet/block/0000000000003d926cb25f3840d58b6f1536c91bae32f6b145699bb8443b64f5) was mined today, with no signalling on bit 2 (actually no signalling except on the bip 320 reserved bits).

Probably doesn't make sense to use future heights for testnet.

Reply:
Ah that's unfortunate. Maybe it should just be always active? Gonna try seeing if I can sync a testnet node with that.

Reply:
I was thinking just add activation rules for testnet after it's active on mainnet, maybe even after it's buried on mainnet?

Reply:
ngl feels like we should abandon testnet3 really.

Reply:
Agree with AJ's suggestion to activate on testnet once active on mainnet given Taproot is available for testing on signet. The fewer heights we have to keep adjusting the better.

Reply:
Dropped these testnet params and marked it as TBD.